modules/ftrackIdSystem.js

```diff
@@ -48,20 +48,36 @@ export const ftrackIdSubmodule = {
    *   similar to the module name and ending in id or Id
    */
   decode (value, config) {
-    if (!value) { return }
-    const ext = {}
+    value = value || {};
```

Comment:
I think it's better to keep the short circuit (`if (!value) return`). From what I understand `value` should always be present here, but if it isn't, it's not correct to default to `{}`: it will cause this logic to return an incomplete object, which would then be translated to essentially gibberish in `eids`.

If this returns undefined, the caller will filter it out from the userID map.

Reply:
Thanks @dgirardi reverted this code